docs/03-tutorials/00-application-connectivity/ac-04-register-secured-api.md

```diff
@@ -2,7 +2,7 @@
 title: Register a secured API
 ---
 
-Application Connectivity allows you to register a secured API exposed by your external solution. The supported authentication methods are [Basic Authentication](https://tools.ietf.org/html/rfc7617), [OAuth](https://tools.ietf.org/html/rfc6750), and client certificates.
+Application Connectivity allows you to register a secured API exposed by your external solution. The supported authentication methods are [Basic Authentication](https://tools.ietf.org/html/rfc7617), [OAuth](https://tools.ietf.org/html/rfc6750), [OAuth 2.0 Mutual-TLS](https://datatracker.ietf.org/doc/html/rfc8705), and client certificates.
```

Comment:
```suggestion
Application Connectivity allows you to register a secured API exposed by your external solution. The supported authentication methods are [Basic Authentication](https://tools.ietf.org/html/rfc7617), [OAuth](https://tools.ietf.org/html/rfc6750), [OAuth 2.0 Mutual TLS](https://datatracker.ietf.org/doc/html/rfc8705), and client certificates.
```
I'd personally go with `mTLS` though, instead of `Mutual TLS`. It's visually immediately more recognizable and I suppose that if someone searches for it, they'd type in `mTLS`, not the full name.

Reply:
I'm OK with that. Please feel free to change it.